core/services/aws/cognitoTrigger/lambdas/customEmailTemplate.ts

```diff
@@ -0,0 +1,24 @@
+import { CustomMessageTriggerEvent, Context, Callback } from 'aws-lambda'
+
+function customEmailTemplateLambda(
+  event: CustomMessageTriggerEvent,
+  _: Context,
+  callback: Callback<CustomMessageTriggerEvent>
```

Comment:
According to [AWS docs](https://docs.aws.amazon.com/lambda/latest/dg/nodejs-handler.html#nodejs-handler-callback), we should consider using async/await instead of callbacks.